api/db-cdr/generate-cdr/bq-schemas/unit_map.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "mode":"REQUIRED",
+    "name": "unit_concept_id",
+    "type": "INTEGER"
+  },
+  {
+    "mode":"REQUIRED",
+    "name": "unit_source_value",
+    "type": "STRING"
+  }
```

Comment:
For later to think about --  I'm tempted to add a preferred_name flag here that would flag this unit_source_value as the preferred name.  But that is another story.

Reply:
Good idea to try.